src/PdfReader/index.tsx

```diff
@@ -30,26 +33,48 @@ function pdfReducer(state: PdfState, action: PdfReaderAction): PdfState {
         ...state,
         isScrolling: action.isScrolling,
       };
+
+    case 'SET_RESOURCE':
+      return {
+        ...state,
+        resource: action.resource,
+      };
+
+    case 'SET_PAGENUM':
+      return {
+        ...state,
+        pageNumber: action.pageNum,
+      };
   }
 }
 
 export default function usePdfReader(args: ReaderArguments): ReaderReturn {
   const { webpubManifestUrl, manifest } = args ?? {};
-  const [state] = React.useReducer(pdfReducer, {
+
+  const [state, dispatch] = React.useReducer(pdfReducer, {
     type: 'PDF',
     colorMode: 'day',
     isScrolling: false,
     fontSize: 16,
     fontFamily: 'sans-serif',
+    resource: '',
+    pageNumber: 1,
   });
 
+  const [numPages, setNumPages] = useState(0);
+  const [isLoading, setLoading] = useState(true);
+
   // initialize the pdf reader
   React.useEffect(() => {
+    console.log('initializing', manifest);
     // bail out if there is not manifest passed in,
     // that indicates that this format is inactive
     if (!manifest) return;
+    const resource: string = manifest.readingOrder![0].href;
```

Comment:
I wouldn't force unwrap this, but probably throw an error if the reading order has nothing in it, which we can later handle further up the component tree